www-dev/index.html

```diff
@@ -14,14 +14,24 @@
         var frame = '<iframe id="chat" src="chat.html{{params}}" style="height: calc(100% - 23px); width: calc(100% - 20px); padding: 10px; border:none;"></iframe>';
         
         function auth() {
+            var pass;
+            var chan;
             var user = prompt("Please enter your username, or leave blank to use without logging in");
             if (!user) {
-                document.body.innerHTML += frame.replace('{{params}}', '');
-                return;
+                chan = prompt("If you want to go into a channel other than global (ie, if you're going to be doing testing that could potentially disturb chat), please type it here (you might want to use \"test\" for such a case). Otherwise, just click ok. (In global you can only look at chat, but in other channels (such as \"test\") you will be able to write)");
+                if(chan && chan != "global"){
+                    user = "GuestForTest";
```

Comment:
I'm thinking it might be a good idea to append a random number (maybe 4 digits) in case there is already a user with this username - doubt it matters if they happen to be the same. I'm open to discussion on this though.